source/troubleshooting.txt

```diff
@@ -201,6 +201,25 @@ Alert Config Not Deleted
 This error might appear if the {+atlas-cli+} can't delete the alert
 configuration specified by the ID. 
 
+podman not found
+~~~~~~~~~~~~~~~~
+
+This error appears if you try to run an ``atlas deployments`` command
+inside our official docker container, ``mongodb/atlas`` in v1.26 or
+later. Instead, follow the steps decribed in
+:ref:`atlas-cli-deploy-docker`.
+
+You should also inspect past containers and remove any that start with
+``mongod`` or ``mongot`` in your {+cluster+} with the following command:
+
+.. code-block::
+
+   podman ps
+
+.. tip::
+
+   You can safely uninstall podman if you're on MacOS.
```

Comment:
if cli was installed via homebrew we can give the command
```
brew uninstall podman
```

Reply:
Could we also address this comment?

Reply:
I added it in the last commit. Is there a clearer way to mention this?

Reply:
@andreaangiolillo I added it in the last commit. Is there a clearer way to mention this?

Reply:
Sorry I missed it. Thanks!